### PR TITLE
Drive mode

### PR DESCRIPTION
**Pull request by Sud-Puth**
Enabled 'drive mode' to control the BB8 from the keyboard input!

### CODE REVIEW

File: commands/drive.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  console.log('Press ctrl+c to exit');
+
+  var stdin = process.stdin;
+  stdin.setRawMode(true);
+  stdin.resume();
+  stdin.setEncoding('utf8');
+
+  stdin.on('data', function(key){
+    bb8.color('#000000');
+    if (key == '\u001B\u005B\u0041') {
```
Comment (shaunbent) [inline commands/drive.js:16]: These key references are pretty ambiguous and its not clear what they are. Maybe we could use the [keypress](https://github.com/TooTallNate/keypress) library instead to make it clearer which key is which.

File: commands/drive.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      //console.log('up');
+      bb8.stop();
+      bb8.roll(150, 0);
+    }
+    if (key == '\u001B\u005B\u0043') {
```
Comment (shaunbent) [inline commands/drive.js:21]: Should probably use `===`: http://stackoverflow.com/a/359509

**Comment by shaunbent** [comment]
It might also be worth factoring in calibration mode, so that you can toggle calibration and on off.

http://sdk.sphero.com/community-apis/javascript-sdk/#custom-commands-startcalibration-callback

**Comment by Sud-Puth** [comment]
Now to think of it. You maybe right. This was just like a PoC.... Lemme close this pull request and work on it when I get some time...

**Pull request closed without merging**